### PR TITLE
Update kubeconfig documentation

### PR DESCRIPTION
**Pull request by aojea**
follow up on #1029

### CODE REVIEW

File: site/content/docs/user/quick-start.md [hunk 1]
```diff
@@ -97,13 +101,11 @@ kind
 kind-2
 ```
 
-Both of these clusters will have a kubeconfig file to go along with them:
+In order to interact with an specific cluster, you only need to specify the
```
Comment (amwat) [inline site/content/docs/user/quick-start.md]: nit: **a** specific cluster

File: site/content/docs/user/quick-start.md [hunk 2]
```diff
@@ -97,13 +101,11 @@ kind
...
+cluster name as a context in kubectl:
 ```
-kind get kubeconfig-path
-/home/user/.kube/kind-config-kind
-
-kind get kubeconfig-path --name kind-2
-/home/user/.kube/kind-config-kind-2
+kubectl cluster-info --context kind
```
Comment (amwat) [inline site/content/docs/user/quick-start.md]: these will now be `kind-kind`

File: site/content/docs/user/quick-start.md [hunk 3]
```diff
@@ -412,3 +414,4 @@ kind, the Kubernetes cluster itself, etc.
 [Private Registries]: /docs/user/private-registries
 [customize control plane with kubeadm]: https://kubernetes.io/docs/setup/independent/control-plane-flags/
 [docker enable ipv6]: https://docs.docker.com/v17.09/engine/userguide/networking/default_network/ipv6/
+[access multiple clusters] https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/
```
Comment (amwat) [inline site/content/docs/user/quick-start.md]: missing `:` after `[access multiple clusters]`

**Comment by BenTheElder** [comment]
IMO this needs to be held until we cut a stable release.

**Comment by aojea** [comment]
/hold

NOTE: Don't forget to merge before the release

**Comment by BenTheElder** [comment]
merged just after the release! 😅

**Pull request merged**